app/code/Magento/ImportExport/Controller/Adminhtml/ImportResult.php

```diff
@@ -68,7 +68,7 @@ protected function addErrorMessages(
             $message = '';
             $counter = 0;
             foreach ($this->getErrorMessages($errorAggregator) as $error) {
-                $message .= ++$counter . '. ' . $error . '<br>';
+                $message .= ++$counter . '. ' . $error . '</br>';
```

Comment:
Hi @irshadcedcoss. Thank you for your contribution. Could you explain the purpose of the `</br>` tag, please?
There are two common versions of the br tag: `<br>` (HTML), `<br />` (XHTML). But not sure about your version.

Thanks.